Source/WebCore/layout/formattingContexts/inline/display/InlineDisplayLineBuilder.cpp

```diff
@@ -124,30 +124,30 @@ InlineDisplayLineBuilder::EnclosingLineGeometry InlineDisplayLineBuilder::collec
 InlineDisplay::Line InlineDisplayLineBuilder::build(const LineLayoutResult& lineLayoutResult, const LineBox& lineBox, bool lineIsFullyTruncatedInBlockDirection) const
 {
     auto& constraints = this->constraints();
-    auto& rootInlineBox = lineBox.rootInlineBox();
     auto isLeftToRightDirection = lineLayoutResult.directionality.inlineBaseDirection == TextDirection::LTR;
-    auto lineBoxLogicalRect = lineBox.logicalRect();
-    auto lineBoxVisualLeft = isLeftToRightDirection
+
+    InlineRect lineBoxLogicalRect = lineBox.logicalRect();
```

Comment:
`    auto lineBoxLogicalRect = lineBox.logicalRect();`

Reply:
I wanted to make it more explicit that we're taking a copy and not a reference here.